nat-lab/tests/test_pq.py

```diff
@@ -410,18 +411,44 @@ async def test_pq_vpn_silent_pq_upgrader(
         client, *_ = env.clients
 
         wg_server = config.WG_SERVER  # use non PQ server
+
```

Comment:
What do you think about having this test also check that `DISCONNECTING` is being emitted?

Reply:
You might be surprised but the call [`disconnect_from_vpn()`](https://github.com/NordSecurity/libtelio/pull/1006/files#diff-d71adf294c60fb47c1fdaabf09dfb47c83a4566748a530105ca35e310f7f49fdR451) below does the check inside